src/app/controllers/user/user.controller.ts

```diff
@@ -1,10 +1,62 @@
 module webShoes {
+  'use strict';
```

Comment:
@guilhermemamprin these warnings will disappear if we stop using strict mode, am I right?
![image](https://cloud.githubusercontent.com/assets/8186616/16770921/152b09c0-4825-11e6-90c8-774515c25bde.png)